ngraph/test/CMakeLists.txt

```diff
@@ -665,5 +665,6 @@ if (NGRAPH_PDPD_FRONTEND_ENABLE AND PDPD_PYTHON_OK)
     add_custom_target(pdpd_test_models DEPENDS ${OUT_FILES})
     add_dependencies(unit-test pdpd_test_models)
     add_dependencies(unit-test paddlepaddle_ngraph_frontend)
+    target_link_libraries(unit-test PRIVATE cnpy)
```

Comment:
this shall be outside of 'if' as pdpd-independent shared/op_fuzzy.cpp includes <cnpy.h>
Also please correct PDPD_FOUND_OK to paddlepaddle_FOUND on line 646

Reply:
Do you mean to use `paddlepaddle_FOUND` instead of `PDPD_PYTHON_OK` ?